sotodlib/tod_ops/pca.py

```diff
@@ -215,3 +215,42 @@ def get_trends(tod, remove=False, size=1, signal=None):
     if remove:
         add_model(tod, trends, scale=-1, signal=signal)
     return trends
+
+
+def get_common_mode(
+    tod,
+    signal_name='signal',
+    method='median',
+    wrap_name=None,
+    weights=None,
+):
+    """Returns common mode timestream between detectors.
```

Comment:
Mention in the description that this uses method 'median' or 'average' across detectors as opposed to a principle component analysis to get the common mode as this diverges from the rest of the module.

Also we are trying to get rid of the `signal_name` implementation everywhere so that these functions could be used with any np array with the right shape even it's not wrapped in aman. Currently the only thing holding us up from that is some baked in dependence in the `fourier_filters` module however once we fix that we're planning to get rid of `signal_name` completely. Since you don't use `fourier_filters` anywhere here I think this implementation example will work replacing `signal_name` with `signal` and allowing it to None, str, or ndarray:

https://github.com/simonsobs/sotodlib/blob/5c26756127403607358e55fbb0b642ab3a0bb523/sotodlib/hwp/hwp.py#L536-L547

Reply:
Addressed this suggestion. Also found typo in this example and fixed it https://github.com/simonsobs/sotodlib/pull/920/commits/7010a6eea7e4813ad22d45b93168278a7bf6a51f